Amplify/Categories/DataStore/Model/Schema/ModelSchema+Attributes.swift

```diff
@@ -16,4 +16,7 @@ public extension ModelSchema {
         attributes.contains(.isSystem)
     }
 
+    var isAuthEnabled: Bool {
+        return !authRules.isEmpty
+    }
```

Comment:
I'm curious as to what `isAuthEnabled` refers to.  Is it simply to say that the models use an `@auth` directive?  If so, then using "Enabled" might be misleading.  Perhaps a better name would be `isAuth`?

Reply:
Interesting feedback, maybe `hasAuth`?

Reply:
i was thinking it means these "models are enabled with auth rules for finer grain control", which means that they require auth, or an authenticated signed request when syncing to the cloud. By having auth directive on the model, they can control how the service checks for access, whether that is checking the token or signed request. 

from the two `isAuth` and `hasAuth`, i think isAuth makes me think of "the model is authenticated" which doesn't make a lot of sense, maybe "hasAuth" is "hasAuthentication" or "hasAuthenticationRules" which is closer to the check `!authRules.isEmpty`

Reply:
`hasAuthenticationRules` lgtm, I in favor of verbosity to communicate clear intent